package.json

```diff
@@ -19,6 +20,7 @@
     "@babel/preset-env": "7.10.2",
     "@storybook/addon-knobs": "^5.3.18",
     "@storybook/addon-storysource": "^5.3.18",
+    "@storybook/storybook-deployer": "^2.8.6",
```

Comment:
@openjck On 2nd thought, perhaps this makes more sense as part of the github actions installation step? Unless you think some devs would want to deploy manually, I can move it there.

Reply:
That works for me. I don't expect we would deploy it manually very often if ever. The only other benefit I could imagine of listing it here is that we could use dependabot to keep it up-to-date, but if there's another way of accomplishing that, that's fine with me.